### DISCUSSION TITLE
My wrenbtc transaction is stuck.

### DISCUSSION THREAD

**Discussion by svaidya413** [wormhole-foundation/wormhole#427] [Problems / bugs] [answered]
I was trying to move wrenbtc from Solana to Ethereum and my transaction is stuck for a while now. MetaMask was not prompt to sign the transaction.

[https://www.wormholebridge.com/#/move?from=SOL&token=RENBTC](url)

wrenbtc:0.996382

**Comment by leoluk** [comment, marked as answer]
Please provide a transaction
